tests/integration/metrics/api/views/test_alerts.py

```diff
@@ -88,7 +88,73 @@ def test_retrieve_returns_400_response(
         assert response.status_code == HTTPStatus.BAD_REQUEST
 
 
-class TestHeatAlertsView:
+class TestColdAlertsView:
+    @property
+    def path(self) -> str:
+        return reverse("cold-alerts-list")
+
+    @pytest.mark.django_db
+    def test_list_returns_correct_response(self):
+        """
+        Given a valid request to the `alerts` endpoint
+        When the `GET /api/alerts/v1/cold` endpoint is hit.
+        Then the expected response is returned.
+        """
+        # Given
+        client = APIClient()
+
+        # When
+        response: Response = client.get(path=self.path)
+
+        # Then
+        assert response.status_code == HTTPStatus.OK
+        assert response.headers["Content-Type"] == "application/json"
+
+    @pytest.mark.django_db
+    @mock.patch.object(GeographiesForAlertsSerializer, "data")
+    @mock.patch.object(GeographiesForAlertsSerializer, "is_valid")
+    def test_retrieve_returns_correct_response(
+        self, mock_is_valid: mock.Mock(), mock_data: mock.Mock()
```

Comment:
```suggestion
        self, mock_is_valid: mock.MagicMock, mock_data: mock.MagicMock
```
These are types not instantiated objects so we don't need to call thm with `()`.
And these are `MagicMock` not `Mock`. The difference being that `MagicMock` implements dunder methods